src/Momento.Sdk.Incubating/SimpleCacheClient.cs

```diff
@@ -818,6 +818,117 @@ public async Task<CacheSetDeleteResponse> SetDeleteAsync(string cacheName, strin
         return await this.dataClient.SetDeleteAsync(cacheName, setName);
     }
 
+    /// <summary>
+    /// Pushes multiple values to the beginning of a list.
```

Comment:
Can we rephrase this to not use the word "add" instead of "push" and also clarify the resulting list will have the new items in the identical order they were passed in?

The motivation for this API was that other implementations of "push" do a `reduce`, which means the order is flipped when adding to the front (ie pushing `[a b c]` to the front of the list produces `[c b a]`).